examples/src/main/scala/com/http4s/rho/swagger/demo/Main.scala

```diff
@@ -19,10 +23,13 @@ object Main extends IOApp {
   logger.info(s"Starting Swagger example on '$port'")
 
   def run(args: List[String]): IO[ExitCode] = {
+    val blockingExecutor = Executors.newCachedThreadPool()
+    implicit val blocker: Blocker = Blocker.liftExecutionContext(ExecutionContext.fromExecutor(blockingExecutor))
```

Comment:
I know this is just the example project, but maybe we should set a good example by wrapping the execution context into a Resource that properly closes the execution context when we are done using it?

Reply:
On the JVM you can also create a blocker by `Blocker[F]`. 
https://github.com/typelevel/cats-effect/blob/ac3644d4afde1ee40c36f0449bdc4f202316889a/core/jvm/src/main/scala/cats/effect/internals/BlockerPlatform.scala#L29

Reply:
:+1: I support every change that makes the example more "production ready". Examples tend to end up being copied to real projects and run on production.